src/libraries/Directory.Build.targets

```diff
@@ -97,6 +105,10 @@
       <RuntimePath>$(ASPNETCoreAppPackageRuntimePath)</RuntimePath>
       <RefPath>$(ASPNETCoreAppPackageRefPath)</RefPath>
     </BinPlaceTargetFrameworks>
+    <BinPlaceDir Include="$(ASPNETCoreAppPackageRootPath)\analyzers\dotnet\$(AnalyzerLanguage)"
+                 Condition="'$(IsAspNetCoreAppAnalyzer)' == 'true'" />
+    <BinPlaceDir Include="$(MicrosoftNetCoreAppRefPackDir)analyzers\dotnet\$(AnalyzerLanguage)"
+                 Condition="'$(IsNETCoreAppAnalyzer)' == 'true'" />
```

Comment:
Why not add support for analyzers in the binplacing targets similar to how I added support for binplacing native assets via `BinPlaceNative`: https://github.com/dotnet/arcade/blob/6b9d24236d8d1906284e6cb6c28e3fe93a69b7d2/src/Microsoft.DotNet.Build.Tasks.TargetFramework.Sdk/src/build/BinPlace.targets#L135?

Reply:
We don’t need the TargetFramework information at all so we bypass that selection and just leverage the copy functionality.

Reply:
https://github.com/dotnet/arcade/pull/7578

Reply:
Why do code generators target netstandard2.0 and not NetCoreAppCurrent?

Reply:
Because they run in the compiler process which runs on both .NETFramework and .NETCore.